apps/web/src/pages/templates/components/ListProviders.tsx

```diff
@@ -79,20 +84,60 @@ export const ListProviders = ({
                 setConfigureChannel(provider.channel);
               }}
             >
-              <Group position="apart">
-                <img
-                  src={'/static/images/providers/' + colorScheme + '/' + provider.logoFileName[`${colorScheme}`]}
-                  alt={provider.displayName}
-                  style={{
-                    height: '24px',
-                    maxWidth: '140px',
-                    opacity: provider.active ? 1 : colorScheme === 'dark' ? 0.4 : 1,
-                  }}
-                />
-                <Text color={provider.active ? colors.success : colors.B60}>
-                  <When truthy={provider.active}>Active</When>
-                  <When truthy={!provider.active}>Disabled</When>
-                </Text>
+              <Group spacing={16} position="apart">
+                <Group spacing={16} position="apart">
+                  <img
+                    src={'/static/images/providers/' + colorScheme + '/square/' + provider.providerId + '.svg'}
+                    alt={provider.displayName}
+                    style={{
+                      height: '24px',
+                      maxWidth: '140px',
+                      opacity: provider.active ? 1 : colorScheme === 'dark' ? 0.4 : 1,
+                    }}
+                  />
+
+                  <Stack
+                    sx={{
+                      width: isIntegrationsListPageEnabled ? '132px' : undefined,
```

Comment:
This was the only way I could get the provider name truncated. Not 100% happy with it so if someone else have a better way of solving it, please tell me :)

Reply:
Could <Text row={1} ..> work here? Text from our design system

Reply:
I tried it and the problem is that it does not create two rows instead it breaks between the text and integration environment pill.